CPR/CprServiceInterface.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of aakb/kontrolgruppen-core-bundle.
+ *
+ * (c) 2019 ITK Development
+ *
+ * This source file is subject to the MIT license.
+ */
+
+namespace Kontrolgruppen\CoreBundle\CPR;
+
+use Kontrolgruppen\CoreBundle\Entity\Client;
+
+interface CprServiceInterface
+{
+    /**
+     * @param int $cpr
+     *
+     * @return array
+     *
+     * @throws CprException
+     */
+    public function find(int $cpr): array;
```

Comment:
What happens if the cpr starts with `0`, e.g. is `03408641234`? Maybe the type should be `string`?